docs/navigate/tools-diagnostics/toc.yml

```diff
@@ -402,8 +402,6 @@ items:
                 href: ../../core/diagnostics/built-in-metrics-runtime.md
               - name: .NET extensions metrics
                 href: ../../core/diagnostics/built-in-metrics-diagnostics.md
-              - name: ASP.NET Core metrics
```

Comment:
Removed the ASP.NET Core metrics link to rely on The Build-in metrics > Overview link instead in the TOC, as the EF metrics does to avoid linking directly to the metrics topic in a different repo and TOC.

Reply:
I think we should keep this TOC entry. The contextual TOC link should look like this:

`href: /aspnet/core/log-mon/metrics/built-in?toc=/dotnet/navigate/tools-diagnostics/toc.json&bc=/dotnet/breadcrumb/toc.json`